pkgs/top-level/appimage-packages.nix

```diff
@@ -0,0 +1,64 @@
+{ appimageTools, stdenv, fetchurl, fetchzip, fetchFromGitHub }:
+
+# fetchzip is for compatibility, bother AppImage builder about that :
+# https://docs.appimage.org/packaging-guide/distribution.html#do-not-put-appimages-into-other-archives
+
+
+let
+  self = appimagePackages;
+  appimagePackages = with self; {
+
+  buildAppImage = {
+    name,
+    version,
+    src,
+    meta,
+    ...
+  } @ attrs:
+    let
+      name = "${attrs.name}-${attrs.version}";
+      attrs' = builtins.removeAttrs attrs ["version"];
+    in appimageTools.wrapType2 (attrs' //  {
+      inherit name src;
+      meta = {
+        license = stdenv.lib.licenses.unfree;
+        platforms = [ "x86_64-linux" ];
+        description = throw "please write meta.description";
+      } // attrs.meta;
+  });
+
+  beakerbrowser = buildAppImage rec {
```

Comment:
Are you using any of these packages? I would prefer if we have a maintainer and actual user who keeps the packages up to date/in a working state.

Reply:
i need examples to make the thing, and i took some that will find maintainer soon, @kamilchm ... or people needs it or colobot i just think was fun to provide as it to children.